docs/content/_docs/query_language.html

```diff
@@ -284,12 +242,30 @@ <h4>Operators</h4>
     <td><pre><code class="language-hql">&lt;a&gt; not in [&lt;b&gt;, ..]</code></pre></td>
     <td><pre><code class="language-json">["not", ["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
   </tr>
+  
+  <tr>
+    <th scope="row">Tag True</th>
+    <td>Returns True</td>
+  </tr>
 
+  <tr>
+    <th scope="row">Tag False</th>
+    <td>Returns False</td>
```

Comment:
We should still have the example here, it just doesn't apply to other filters/tags. So the false filter looks like just `[false]`